Util/ComposerManager.php

```diff
@@ -38,13 +45,13 @@ public function initialize(): void
         $write = false;
 
         if (!array_key_exists('vendor-bin-install', $data['scripts'])) {
-            $data['scripts']['vendor-bin-install'] = '@composer bin all install --ansi';
+            $data['scripts']['vendor-bin-install'] = 'if test -d vendor/protacon/test-tools-bundle; then @composer bin all install --ansi; fi';
```

Comment:
```suggestion
            $data['scripts']['vendor-bin-install'] = 'php -r \'exit(is_dir("vendor/protacon/test-tools-bundle") ? 0 : -1);\' && @composer bin all install --ansi';
```

Reply:
I think that the `test` command will fail in Windows systems. Could this check be done using php?